template.tpl

```diff
@@ -318,6 +318,43 @@ ___TEMPLATE_PARAMETERS___
         "type": "EQUALS"
       }
     ]
+  },
+  {
+    "type": "GROUP",
+    "name": "additionalInformation",
+    "groupStyle": "ZIPPY_CLOSED",
```

Comment:
The section title here is "Additional Information", and then there's a nested table for "Product Information".  Will there ever be any other sections within the "Additional Information"?  If not, why not simply make the "Product Information" the whole section?

Reply:
It's a little bit of a chicken and egg situation. I wanted the outer `Additional Information` group so that it could be collapsed by default. But that section isn't properly supporting Help Text, so I added the `Product Information` name to the nested table so that it would display the help text. But I didn't want to have it show `Product Information` twice...